test/api_test.js

```diff
@@ -540,6 +540,40 @@ describe("KintoClient", () => {
     });
   });
 
+  /** @test {KintoClient#listPermissions} */
+  describe("#listPermissions()", () => {
+    const data = [{id: "a"}, {id: "b"}];
+
```

Comment:
I don't if we did for other stuff, but it may be worth adding a test to check that it fails nicely when capability is not present ?